### PR TITLE
Remove tech jargon from photo picker related content

### PR DESCRIPTION
**Pull request by RitikaPahwa4444**
Removed tech jargon, and added details about the 'Browse' option to open the system file picker within the new Android picker.

### CODE REVIEW

File: docs.html [hunk 1]
```diff
@@ -115,61 +115,73 @@ <h2 id="get-content"><a href="#get-content">What is GET_CONTENT?</a></h2>
...
                 </li>
             </ul>
         </p>
 
-        <h4>How is the new GET_CONTENT picker different from the classic picker?</h4>
-        <p>Android recently introduced a new photo picker for devices having Android 11 or higher. Unfortunately, this photo picker removes location tags from the EXIF metadata of images. The new photo picker takes over GET_CONTENT on some devices. This means you will see the new photo picker and your pictures might lose location information in the EXIF metadata.</p> 
-        <p>In order to preserve location information, we decided to go with the system file picker by default for the classic picker and disabled the GET_CONTENT picker.</p>
+        <h4>How is the new Android picker different from the Document based picker?</h4>
+        <p>Android recently introduced a new photo picker for devices having Android 11 or higher. Unfortunately, this photo picker removes location tags from the EXIF metadata of images.</p> 
+        <p>In order to preserve location information, we decided to go with the system file picker by default for the Document based picker and disabled the new Android picker.</p>
```
Comment (sivaraam) [inline docs.html:142]: > In order to preserve location information, we decided to go with the system file picker by default for the Document based picker and disabled the new Android picker.

I suppose this phrase could be improved a bit. May be tweak it like:

> In order to preserve location information, we decided to go with the Document based picker by default and skipped the new Android picker.

(note: as you know we can't disable the new photo picker)

File: docs.html [hunk 2]
```diff
@@ -115,61 +115,73 @@ <h2 id="get-content"><a href="#get-content">What is GET_CONTENT?</a></h2>
...
         
         <p>To summarize the advantages and disadvantages:</p>
 
-        <p><b>GET_CONTENT Picker:</b>
+        <p><b>New Android Picker:</b>
```
Comment (sivaraam) [inline docs.html:146]: May be clarify that only the "New Android picker with takeover" is problematic ? 

When takeover is not happening, the location conventional GET_CONTENT picker kicks in which does not have location loss issues.

File: docs.html [hunk 3]
```diff
@@ -115,61 +115,73 @@ <h2 id="get-content"><a href="#get-content">What is GET_CONTENT?</a></h2>
...
             <ul>
                 <li><b>Merits:</b> Lists all image providers as marked in red in the figure above, user-friendly interface.</li>
```
Comment (sivaraam) [inline docs.html:148]: I'm kind of unsure. Does this merit really apply to the new photo picker with takeover ? If not, we may need to reconsider the phrasing 🤔

File: docs.html [hunk 4]
```diff
@@ -104,8 +104,8 @@ <h2 id="getting-app-logs-from-android-studio"><a href="#getting-app-logs-from-an
...
-        <h2 id="get-content"><a href="#get-content">What is GET_CONTENT?</a></h2>
-        <p>The Commons app lets you pick what picture(s) you want to upload, using either the camera button or what we call a "picker". The "custom picker" has an icon that includes a small Commons logo, because it has Commons-specific features such as checking whether a picture exists on the Commons server or not. The "normal picker" now has two variants: the "classic picker" and the "GET_CONTENT picker" (you may choose your favourite variant from Settings). To sum up, the app now has 3 pickers:</p>
+        <h2 id="get-content"><a href="#get-content">What is Document based picker?</a></h2>
+        <p>The Commons app lets you pick what picture(s) you want to upload, using either the camera button or what we call a "picker". The "custom picker" has an icon that includes a small Commons logo, because it has Commons-specific features such as checking whether a picture exists on the Commons server or not. The "normal picker" now has two variants: the "Document based picker" and the "new Android picker" (you may choose your favourite variant from Settings). To sum up, the app now has 3 pickers:</p>
```
Comment (sivaraam) [inline docs.html:108]: > and the "new Android picker" ...

In retrospect, it is incorrect to refer to the GET_CONTENT picker as  the "new Android picker" since it has been there for a long time. The actual new picker is one that was recently added and appears in the GET_CONTENT intent when takeover is enabled.

So, I believe it makes sense to distinguish the same so that users could clearly understand the different. Could you kindly go through the content once and rephrase it to distinguish both of them better ? I guess we could use the term "media picker" to refer to the old GET_CONTENT picker and "new media picker" for the one shown when takeover is enabled. What do you think ?

**Review by sivaraam — CHANGES_REQUESTED** [review]
Thanks for the change!

I've left some comments. Kindly take a look at them.

File: docs.html (hunk 2 above)
Comment (RitikaPahwa4444) [reply docs.html:146]: The one without takeover isn't new. Any special name you would like to suggest for that one? We are skipping GET_CONTENT altogether, explaining takeover without it sounds weird.

File: docs.html (hunk 3 above)
Comment (RitikaPahwa4444) [reply docs.html:148]: It always does. I've explained how users can get these providers by using 'Browse'. That's not possible with the document style picker.

File: docs.html (hunk 4 above)
Comment (RitikaPahwa4444) [reply docs.html:108]: Since the dialog you added calls it "new Android picker", I avoid introducing new terms like media picker, or something similar.

File: docs.html (hunk 3 above)
Comment (sivaraam) [reply docs.html:148]: The way the merit is phrased makes it sound as if it applies to new media picker itself, does it not ? I was thinking we might need to tweak it a bit (like making it explicit that the merit applies only to traditional media picker?).

We could also clarify the demerit applies only to new media picker (which appears upon takeover).

File: docs.html (hunk 2 above)
Comment (sivaraam) [reply docs.html:146]: > We are skipping GET_CONTENT altogether, explaining takeover without it sounds weird.

I could understand this. The word "takeover" seems to make little sense when we're trying to keep the content less technical. Could you look into phrasing the content by avoiding the mention of the same (except in the command of course)?

For instance, the following phrase :

> The new Android picker that you may try in the Contributions and Nearby tab, it varies from device to device:
> The new Android picker without takeover
> The new Android picker with takeover
... could be tweaked as ...

> The media picker that you may try in the Contributions and Nearby tab, it varies from device to device:
> The traditional media picker  (does not strip location)
> The new media picker (strips location)

File: docs.html (hunk 4 above)
Comment (sivaraam) [reply docs.html:108]: We could change the dialog text if necessary, Ritika. Let that not be a reason to compromise on the readability / understandability of the text.

File: docs.html (hunk 2 above)
Comment (RitikaPahwa4444) [reply docs.html:146]: Thank you for the example. I'll modify this.

File: docs.html (hunk 3 above)
Comment (RitikaPahwa4444) [reply docs.html:148]: > The way the merit is phrased makes it sound as if it applies to new media picker itself, does it not ?

Yes. No doubt it provides a way to get them via 'Browse'. Moreover, they are planning to extend the support [[source](https://android-developers.googleblog.com/2023/04/photo-picker-everywhere.html)]:
> Later this year, the photo picker will seamlessly support cloud storage providers like Google Photos, allowing users to select their remote content without having to leave your app, and without any code change on the developers side.
Could be modified later once they do.

**Review by sivaraam — APPROVED** [review]

**Comment by sivaraam** [comment]
Looks good. Thanks, Ritika! 🙂
